lib/logger.js

```diff
@@ -0,0 +1,5 @@
+import { getLogger } from 'appium-logger';
+
+const logger = getLogger('appium-doctor');
```

Comment:
can we use `getLogger('AppiumDoctor');` instead? that's the style for logger names we've been using